### DISCUSSION TITLE
Have been using NAPS2 for a while. Good product, but maybe we should update the UX design lol.

### DISCUSSION THREAD

**Discussion by starsang1995** [cyanfish/naps2#148] [Development]
I have been using NAPS2 for 2 while now. The batch scan and other functions are great. But I think it would be better if we can have a more "modern" UX/UI design.

I do understand this is normally not a priority for open source projects, but please let me know if I can be of any assistance improving the designs, say redesign the buttons as a start lol.

**Comment by cloudmaker97** [comment]
I would like to see a more dark version of NAPS2 for the night owls like me 🦉(see #109)
But i like the minimalistic and "feature-first" concept of the current software and would like to keep it.

**Comment by cyanfish** [comment]
Depends what you mean:
- Migrating from WinForms to WPF is [planned](https://github.com/cyanfish/naps2/issues/81) which should feel subtly more modern (and would also allow a dark mode).
- Switching the icon set I'm not really interested in, though if someone had a set of icons they liked I might consider having support for multiple themes.
- Doing something drastic like trying to switch to WinUI3 and match the current Windows design language (such as it is) isn't feasible as Eto doesn't [support it yet](https://github.com/picoe/Eto/issues/2513).

(And though I assume we're all talking about Windows, I should also mention that Mac does have a [modernized](https://www.naps2.com/images/naps2-desktop-mac.png) UI.)
